### DISCUSSION TITLE
a question about database

### DISCUSSION THREAD

**Discussion by hysaint** [oschwengers/bakta#119] [Q&A] [answered]
Dear esteemed author @oschwengers :
 I have one another question.Should the downloaded database folder have a internal database amrfinderplus-db originally. I can not find the subfolder until I run the command amrfinder_update --force_update --database <db-path>/amrfinderplus-db/. Is there any problems in the downloaded database?

**Comment by oschwengers** [comment, marked as answer]
Hi, thanks for reaching out and no, that is the expected behavior. The `amrfinderplus-db` folder is not included in the `db.tar.gz` tarball. Either you download an install the database via `bakta_db install` which takes care of the entire process or you manually download the tarball and subsequently install the AMRFinder db as you've already mentioned:

```
$ wget https://zenodo.org/record/5961398/files/db.tar.gz
$ tar -xzf db.tar.gz
$ rm db.tar.gz
$ amrfinder_update --force_update --database db/amrfinderplus-db/
```

**Reply by hysaint** [reply]
thanks!
